tests/test_xlsx_validation.py

```diff
@@ -0,0 +1,24 @@
+import os
+from unittest import TestCase
+
+from eva_submission.xlsx.xlsx_validation import EVAXlsValidator
+
+
+class TestEVAXlsValidator(TestCase):
+
+    def setUp(self) -> None:
+        brokering_folder = os.path.join(os.path.dirname(__file__), 'resources', 'brokering')
+        metadata_file = os.path.join(brokering_folder, 'metadata_sheet2.xlsx')
+        self.validator = EVAXlsValidator(metadata_file)
+
+    def test_cerberus_validation(self):
```

Comment:
Suggest renaming to **test_schema_validation** since Cerberus is just a library that we use to accomplish schema validation...

Reply:
I like naming the test from the function they are testing. Hence the name.This said these tests are not finished and I still need to create metadata spreadsheets failing the validations.